### ISSUE TITLE
MyKiCadLibs-Lib.lib missing?

### ISSUE THREAD

**Issue by stapelberg** [ohdsp/CoreOne#1]
Hey,

thanks for sharing this project! I am planning an xCORE200-based project, too, so I wanted to see how you did things, but when opening the schematic in Kicad, I get the error message “The following libraries were not found: MyKiCadLibs-Lib.lib”.

**Comment by ohdsp** [comment]
Depends on the KiCad version you are using. This project was created in KiCad 4, but will work in KiCad 5 fine. The readme.md file in this repository has instructions for setting up libraries in KiCad 4 versions. You do also need to download the Kicad-Lib libraries from here:
https://github.com/ohdsp/KiCad-Libs

If you follow the directory structure in readme.md it works fine (on windows machines at least). 

If you are using KiCad 5 generally the same instructions apply. There is a new project - uDSP - this had a readme.md file with instructions for KiCad 5 setups:
https://github.com/ohdsp/uDSP/blob/master/README.md

This should apply to all projects new and old. If you open this project in KiCad 5 it will ask to remap symbols, just click the remap button. I just tried a fresh checkout of this project and the Kicad-Libs, following the directory structure in the readme.md file. When I opened the schematic and KiCad 5 asked to remap I clicked "Remap Symbols" button. It picked up the old library fine and worked as expected.

Regards

Paul
